fdbserver/RestoreMaster.actor.h

```diff
@@ -54,7 +54,7 @@ struct VersionBatch {
 struct RestoreMasterData : RestoreRoleData, public ReferenceCounted<RestoreMasterData> {
 	// rangeToApplier is in master and loader node. Loader uses this to determine which applier a mutation should be sent.
 	//   KeyRef is the inclusive lower bound of the key range the applier (UID) is responsible for
-	std::map<Standalone<KeyRef>, UID> rangeToApplier;
+	std::map<Key, UID> rangeToApplier;
```

Comment:
Do you really want to use `Key` here? `Key` is actually a `StringRef`, which means the underlying memory must be live for the time `rangToApplier` is valid. In contrast, `Standalone` has its own memory buffer.